mypyc/primitives/tuple_ops.py

```diff
@@ -27,6 +28,21 @@
     error_kind=ERR_MAGIC,
     var_arg_type=object_rprimitive)
 
+new_tuple_with_length_op = custom_op(
+    arg_types=[c_pyssize_t_rprimitive],
+    return_type=tuple_rprimitive,
+    c_function_name='PyTuple_New',
+    error_kind=ERR_MAGIC)
+
+# PyTuple_SET_ITEM does no error checking,
+# and should only be used to fill in brand new tuples.
+new_tuple_set_item_op = custom_op(
+    arg_types=[tuple_rprimitive, c_pyssize_t_rprimitive, object_rprimitive],
+    return_type=void_rtype,
+    c_function_name='PyTuple_SET_ITEM',
+    error_kind=ERR_NEVER
+)
+
```

Comment:
Using `PyTuple_SET_ITEM ` instead of `PyTuple_SetItem` since we only use it after initialization.

Reply:
This makes sense. 

Since `PyTuple_SET_ITEM` is a macro, non-C backend won't be able to use it, however (but currently we only have a C back end). It's okay to use it for now, but we'll need a follow-up issue to eventually replace this with lower-level IR operations such as `SetMem`. If you want to see how this is done, `new_list_op` does a similar thing as `PyTuple_SET_ITEM` but for lists, using low-level IR operations only.